### PR TITLE
[no-release-notes] Unskip script tests for `__DOLT_1__`

### PR DESCRIPTION
**Pull request by andrew-wm-arthur**

### CODE REVIEW

File: go/libraries/doltcore/sqle/index/prolly_index_iter.go [hunk 1]
```diff
@@ -365,24 +351,176 @@ func coveringIndexMapping(idx DoltIndex, pkSch sql.PrimaryKeySchema) (keyMap val
...
+	eg.Go(func() error {
+		return iter.queueRows(c)
+	})
+
+	return iter, nil
+}
+
+// Next returns the next row from the iterator.
+func (p prollyKeylessIndexIter) Next(ctx *sql.Context) (r sql.Row, err error) {
+	for {
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/index/prolly_index_iter.go:439]: Why busy wait here? Just read off the channel, no loop or select needed

File: go/libraries/doltcore/sqle/writer/prolly_index_writer.go [hunk 2]
```diff
@@ -169,9 +169,17 @@ func (m prollyIndexWriter) Update(ctx context.Context, oldRow sql.Row, newRow sq
...
 	newKey := m.keyBld.Build(sharePool)
 
-	_, err := m.mut.Has(ctx, newKey)
+	ok, err := m.mut.Has(ctx, newKey)
 	if err != nil {
 		return err
+	} else if ok {
+		// All secondary writers have a name, while the primary does not. If this is a secondary writer, then we can
+		// bypass the keyError() call as it will be done in the calling primary writer.
+		if m.name == "" {
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/writer/prolly_index_writer.go:178]: Probably want a more formal way of determining if this index is primary or not, add a TODO?

**Review by zachmu — APPROVED** [review]
LGTM

File: go/libraries/doltcore/sqle/writer/prolly_index_writer.go (hunk 2 above)
Comment (andrew-wm-arthur) [reply go/libraries/doltcore/sqle/writer/prolly_index_writer.go:178]: good call

**Pull request merged**